inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp

```diff
@@ -641,6 +641,16 @@ void MKLDNNGraphOptimizer::FuseConvolutionAndActivation(MKLDNNGraph &graph) {
     }
 }
 
+static bool BF16QuantizeNodeFusing(MKLDNNNodePtr parentNode, MKLDNNNodePtr childNode) {
+    return childNode->getType() == Quantize &&
+        one_of(Precision::BF16,
+            parentNode->getCnnLayer()->precision,
+            childNode->getCnnLayer()->precision,
+            parentNode->getCnnLayer()->outData[0].get()->getPrecision(),
+            childNode->getCnnLayer()->insData[0].lock()->getPrecision(),
```

Comment:
Why Quantize node input precision is blocker? int8 -> [Conv] -> bf16 -> [FQ] -> fp32 ---> int8 -> [Conv + FQ] -> fp32. Seems like supported case

Reply:
Changed